Lib/test/test_type_params.py

```diff
@@ -1183,11 +1183,16 @@ def func1[X](x: X) -> X: ...
 def func2[X, Y](x: X | Y) -> X | Y: ...
 def func3[X, *Y, **Z](x: X, y: tuple[*Y], z: Z) -> X: ...
 def func4[X: int, Y: (bytes, str)](x: X, y: Y) -> X | Y: ...
+def func3b[X, *Y, **Z]() -> X: return Class3[X, Y, Z]()
+def func5[Baz](): return Class5[Baz]()
 
 class Class1[X]: ...
 class Class2[X, Y]: ...
 class Class3[X, *Y, **Z]: ...
 class Class4[X: int, Y: (bytes, str)]: ...
+class Class5(Generic[T]): pass
+class Class6:
+    def meth[Baz](): return Class5[Baz]()
```

Comment:
Would there be a difference with `Class5[Baz]` and `Class1[Baz]`? as in the first case, `T` is globally available but not in the second?

Reply:
Not that I can think of, changing.